io-manager.js

```diff
@@ -616,37 +624,36 @@ ioManager.keyup = e => {
     }
     case 70: { // F
       // if (cameraManager.pointerLockElement) {
-        ioManager.keys.forward = false;
+      ioManager.keys.forward = false;

```

Comment:
These spacing changes around commented code sections are not an improvement. Can we remove?

Either that, or the comments should also be removed.